public/app/features/folders/state/actions.ts

```diff
@@ -28,7 +28,9 @@ export function saveFolder(folder: FolderState): ThunkResult<void> {
     });
 
     dispatch(notifyApp(createSuccessNotification('Folder saved')));
-    locationService.push(`${res.url}/settings`);
+    dispatch(loadFolder(res));
+    const resUrl = config.appSubUrl ? `${res.url}`.replace(new RegExp(`^${config.appSubUrl}`), '') : res.url;
```

Comment:
Instead of this we could do the same that is done in `createNewFolder` [here](https://github.com/grafana/grafana/pull/60721/files#diff-b9e7ae94007615f9ed5a8a0bcfaf3856d828e1b2a7fb59c93456209ac0cc4624R153), there seems to be a helper function that already does this 😄

Reply:
Yes, It makes sense to use the helper function. Thanks